pkg/controller/disruption/disruption.go

```diff
@@ -21,9 +21,8 @@ import (
 	"fmt"
 	"time"
 
-	apps "k8s.io/api/apps/v1beta1"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/api/extensions/v1beta1"
+	apps "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
```

Comment:
this should be technically `corev1` but since it wasn't changed I won't ask for it to be fixed here